libs/server/repository/api-rest/src/lib/controllers/statistics.controller.ts

```diff
@@ -1,43 +1,56 @@
-import { Controller, Get, Param } from '@nestjs/common';
+import { Controller, Get, Param, UseGuards, Res } from '@nestjs/common';
 import { ApiOkResponse, ApiTags } from '@nestjs/swagger';
+import { Response } from 'express';
 import {
   RepositoriesStatisticsItemReadModel,
   RepositoryFacade,
   ReviewersStatisticsItemReadModel,
   ReviewerStatisticsReadModel
 } from '@pimp-my-pr/server/repository/core/application-services';
 import { ListRepositoriesResponse, UserStatistics } from '@pimp-my-pr/shared/domain';
+import { AuthGuard } from '@pimp-my-pr/server/auth/public';
 
 @ApiTags('statistics')
 @Controller('statistics')
+@UseGuards(AuthGuard)
 export class StatisticsController {
   constructor(private repositoryFacade: RepositoryFacade) {}
 
   @ApiOkResponse({ type: [RepositoriesStatisticsItemReadModel] })
   @Get('repository')
-  list(): Promise<ListRepositoriesResponse> {
-    return this.repositoryFacade.listRepositoriesStatistics();
+  list(@Res() res: Response): Promise<ListRepositoriesResponse> {
+    return this.repositoryFacade.listRepositoriesStatistics(res.locals.token, res.locals.platform);
   }
 
   @ApiOkResponse({ type: [RepositoriesStatisticsItemReadModel] })
   @Get('repository/:repositoryId')
   listSingleRepository(
-    @Param('repositoryId') repositoryId: string
+    @Param('repositoryId') repositoryId: string,
+    @Res() res: Response
   ): Promise<ListRepositoriesResponse> {
-    return this.repositoryFacade.getRepositoryStatistics({ repositoryId });
+    return this.repositoryFacade.getRepositoryStatistics(
+      { repositoryId },
```

Comment:
We need to decide how we will pass arguments. By a "payload" object or one argument by another.